sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DistroHelper.java

```diff
@@ -555,6 +579,38 @@ public Properties getArtifactProperties(Artifact artifact, File directory, Strin
 		return properties;
 	}
 
+	// TODO: This needs unit tests, but to do so will require more refactoring to allow downloadDistroProperties to be mocked.
```

Comment:
Per my comment, this needs to have proper tests around it, but I'm punting that to a later date due to the potential complexity of doing this in a unit test and needing to mock methods within the class being tested.